ionoscloud/resource_lan.go

```diff
@@ -252,13 +254,24 @@ func resourceLanDelete(ctx context.Context, d *schema.ResourceData, meta interfa
 		return diag.FromErr(err)
 	}
 
-	apiResponse, err := client.LANsApi.DatacentersLansDelete(ctx, dcId, d.Id()).Execute()
-	logApiRequestTime(apiResponse)
-
+	err := retry.RetryContext(ctx, d.Timeout(schema.TimeoutDelete), func() *retry.RetryError {
```

Comment:
@cristiGuranIonos can we simulate a test for this situation or is it hard to reproduce?

Reply:
the tests that will pass for inmemorydb and nfs will mean this works. Probably only inmemorydb as nfs has another issue on delete also.